src/Sulu/Component/Webspace/Manager/WebspaceCollectionBuilder.php

```diff
@@ -100,8 +101,8 @@ public function build()
         $this->portals = array();
         $this->portalInformations = array();
 
+        /** @var SplFileInfo $file */
```

Comment:
No problem to put it back. I've done it, because I've never seen it before this way and even PhpStorm puts the doc above the variable, if you issue `Alt + Enter -> Add PHPDoc comment` on the variable.

:smiley: